.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: ruby
+rvm:
+  - 2.2
+sudo: false
+cache:
+  bundler: false
+  directories:
+    - disque
+    - gems
+before_install:
+  - test -d disque/.git || git clone -n https://github.com/antirez/disque.git
```

Comment:
It's probably slightly faster to just download the master's tar archive, since we don't care at all about any git-ty features here.